src/Cameras/VR/babylon.vrExperienceHelper.ts

```diff
@@ -138,6 +148,22 @@ module BABYLON {
         }
 
         public _setLaserPointerParent(mesh:AbstractMesh){
+            var makeNotPick = (root: AbstractMesh) => {
+                root.name += " laserPointer";
+                root.getChildMeshes().forEach((c) => {
+                    makeNotPick(c);
+                });
+            }
+            makeNotPick(mesh);
+            var childMeshes = mesh.getChildMeshes();
+
+            for (var i = 0; i < childMeshes.length; i++) {
+                if (childMeshes[i].name && childMeshes[i].name.indexOf("POINTING_POSE") >= 0) {

```

Comment:
Would be better to have POINTING_POSE stored somewhere as  a (public, readonly? ) static variable

Reply:
Works for me, added